src/dotnet/commands/dotnet-migrate/Program.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Microsoft.DotNet.Cli.CommandLine;
+using Microsoft.DotNet.Cli.Utils;
+
+namespace Microsoft.DotNet.Tools.Migrate
+{
+    public partial class MigrateCommand
+    {
+        public static int Run(string[] args)
+        {
+            DebugHelper.HandleDebugSwitch(ref args);
+
+            CommandLineApplication app = new CommandLineApplication(throwOnUnexpectedArg: false);
+            app.Name = "dotnet migrate";
+            app.FullName = ".NET Migrate Command";
+            app.Description = "Command used to migrate project.json projects to msbuild";
+            app.HandleResponseFiles = true;
+            app.AllowArgumentSeparator = true;
+            app.HelpOption("-h|--help");
+
+            CommandOption template = app.Option("-t|--template-file", "Base MSBuild template to use for migrated app. The default is the project included in dotnet new -t msbuild", CommandOptionType.SingleValue);
+            CommandOption output = app.Option("-o|--output", "Directory to output migrated project to. The default is the project directory", CommandOptionType.SingleValue);
```

Comment:
This option is intended to allow you to do a migration that doesn't mutate artifacts in-place. 

I see what you're saying though, we need to make a recursive copy of the project directory to the output directory. Will fix.